modules/ROOT/pages/index.adoc

```diff
@@ -48,11 +49,9 @@ image::os-use-object-store-v2.png[Click Use Object Store V2 in the Runtime Manag
 * The Object Store Connector works for both Object Store version 1 and 2.
 * Object Store v2 enforces a maximum data persistence of 30 days from time of creation.
 * Accounts that have not purchased the Object Store v2 premium SKU are rate limited to 10 transactions per second per application. To go beyond the base 10 TPS rate limit, contact your MuleSoft account team.
-* In Mule 4, Object Store v2 overrides the `_defaultPersistentObjectStore` object store. No further configuration is required.
-If the connector or component that you are using uses that object store, then you have access to Object Store v2.
+* For CloudHub applications using Mule 4, if you choose Object Store v2, Object Store v2 overrides the `_defaultPersistentObjectStore` object store.
 +
-In Mule 3, Object Store v2 overrides the `_defaultUserObjectStore` object store. In the Mule 3 Object Store connector,
-you must set the Object Store Reference field to `_defaultUserObjectStore` for data to be stored in Object Store v2.
+For CloudHub applications using Mule 3, Object Store v2 overrides the `_defaultUserObjectStore` object store. To use Object Store v2 in CloudHub applications, set the Object Store Reference field to the `_defaultUserObjectStore` value.
```

Comment:
I don't understand this. 
Object Store v2 overrides the `_defaultUserObjectStore` object store reference on CloudHub apps running on Mule 3? 
Second sentence doesn't specify CloudHub apps running on Mule 3, but Im assuming you need to set that object store reference only for Mule 3 apps? Or do you need to do that in all cases if you are using Object Store v2?